plugins/RedditPlugin/__init__.py

```diff
@@ -9,20 +9,14 @@ class RedditPlugin(GarfieldPlugin):
     def __init__(self, manifest, bot):
         super().__init__(manifest, bot)
         
-        # Get login details from env
-        load_dotenv()
-        self.reddit_client_id=os.environ["REDDIT_CLIENT_ID"]
-        self.reddit_client_secret=os.environ["REDDIT_CLIENT_SECRET"]
-        self.reddit_client_username=os.environ["REDDIT_USERNAME"]
-        self.reddit_client_password=os.environ["REDDIT_PASSWORD"]
-        
-        # Login
+        # Login with details from env
+        load_dotenv(find_dotenv())
```

Comment:
This is unneeded, as the dotenv processing outside will populate the environment variables for you already.

Reply:
Good catch.